dor-workflow-service.gemspec

```diff
@@ -17,13 +17,13 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(spec)/})
   gem.require_paths = ["lib"]
 
-  gem.add_dependency "activesupport", '>= 3.2.1', '< 4.2.2'
+  gem.add_dependency "activesupport", '>= 3.2.1', '< 5'
```

Comment:
cool, i think this obviates the need for pull request #9 (which i'll close).  that also had a version bump, but i imagine that'll happen on master directly or get added to this pull request or something?